extensions-contrib/opencensus-extensions/src/main/java/org/apache/druid/data/input/opencensus/protobuf/OpenCensusProtobufReader.java

```diff
@@ -81,29 +97,75 @@ public OpenCensusProtobufReader(
   }
 
   @Override
-  public CloseableIterator<InputRow> read()
+  public CloseableIterator<InputRow> read() throws IOException
```

Comment:
this change is unnecessary if we read the ByteBuffer directly as suggested below